src/main/java/racingcar_tdd/Application.java

```diff
@@ -0,0 +1,57 @@
+package racingcar_tdd;
+
+import racingcar_tdd.domain.Car;
+import racingcar_tdd.domain.Cars;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Application {
+
+    public static void main(String args[]) {
+
+        Scanner input = new Scanner(System.in);
+
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        List<String> carnamelist = Arrays.asList(input.next().split(","));
+
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        int track = input.nextInt();
+
+        var racing_cars = makeCar(carnamelist);
```

Comment:
자바 변수명은 카멜 케이스를 사용합니다.
```suggestion
        var racingCars = makeCar(carnamelist);
```